third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -790,6 +790,15 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		cluster.Network = network.RelativeLink()
 	}
 
+
+	// When parsing a subnetwork by name, we expect region or zone to be set.
```

Comment:
Should we just do this after initial getLocation? I'm sure there's more than one ParseXFieldValue to be had here.

Reply:
Sure!